frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,250 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    button.button.is-small.is-outlined.c-active-button(
+      @click='switchList'
+    )
+      | {{ ephemeral.activeOn? L('Active links') : L('All links') }}
+      i.icon-angle-down.c-arrow
+
+  p.c-invite-description
+    i18n.has-text-1 Here's a list of all invite links you own
+
+  table.table.c-table(v-if='activeList.length')
+    thead
+      tr
+        i18n.name(tag='th') created for
+        i18n.invite-link(tag='th') invite link
+        i18n.state(tag='th') state
+        th.action
+    tbody
+      tr(
+        v-for='(item, index) in activeList'
+        :key='index'
+      )
+        td.name
+          | {{ item.name }}
+          tooltip.c-name-tooltip(
+            direction='top'
+            :isTextCenter='true'
```

Comment:
discuss with @mmbotelho , but I think this share tooltip should be placed bellow to the left, so in small screens it isn't cutted like here:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/14869087/66988376-19991180-f0cb-11e9-9aa1-9aabb1d47589.png">